witness/src/Dscp/Witness/Mempool.hs

```diff
@@ -24,7 +25,10 @@ newMempoolVar :: MonadIO m => m (TVar Mempool)
 newMempoolVar = liftIO $ newTVarIO (Mempool mempty)
 
 addTxToMempool :: MonadIO m => MempoolVar -> GTxWitnessed -> m ()
-addTxToMempool var tx = atomically $ modifyTVar var $ mpTxs %~ ((:) tx)
+addTxToMempool var tx = atomically $ modifyTVar var $ mpTxs %~ (:) tx
```

Comment:
Up to me, it was not that bad previously, you could see at which side of operator operand lies at the first glance.

Reply:
Emm, there is only one operator in `mpTxs %~ (:) tx` expression. The `(:)` is a full section and is syntaxically a function.